test/Mono.Linker.Tests.Cases/Reflection/ExpressionFieldString.cs

```diff
@@ -22,5 +25,57 @@ public static void Main ()
 
 		[Kept]
 		private int TestName1;
+
+		[Kept]
+		static string UnknownString ()
+		{
+			return "unknownstring";
+		}
+
+		[Kept]
+		static void Branch_SystemTypeValueNode_KnownStringValue_NonStatic ()
+		{
+			var expr = Expression.Field (Expression.Parameter (typeof (int), "somename"), typeof (ExpressionFieldString), "TestName1");
```

Comment:
Similar comment to `Call` - test the ability to process multiple potential values for name and type.